app/queries/get_fraud_matches.rb

```diff
@@ -3,38 +3,38 @@ def self.call
     ActiveRecord::Base.connection.exec_query(
       "SELECT DISTINCT application_details.candidate_id,
           application_details.first_name,
-          application_details.last_name,
+          application_details.last_name last_name,
           TRIM(UPPER(application_details.postcode)) postcode,
```

Comment:
```suggestion
          REPLACE(UPPER(application_details.postcode), ' ', '') postcode,
```
for consistency with matching transform

Reply:
I thought it might be nicer to do the comparison properly but show the actual value as it is in the db for clarity.

Happy to defer to what you think is best though?

Reply:
I think you might be right, I was getting a bit confused about the `DISTINCT` and thinking we had to _pick one_ of the duplicate values but that's not the case. Leave as is!